keyboards/southpaw75/southpaw75.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT( \
+	K00, K01, K02, K03, K04, K05, K06, K07, K08, \
+	K10, K11, K12, K13, K14, K15, K16, K17, K18, \
+	K20, K21, K22, K23, K24, K25, K26, K27, K28, \
+	K30, K31, K32, K33, K34, K35, K36, K37, K38, \
+	K40, K41, K42, K43, K44, K45, K46, K47, K48, \
+	K50, K51, K52, K53, K54, K55, K56, K57, K58, \
+	K60, K61, K62, K63, K64, K65, K66, K67, K68, \
+	K70, K71, K72, K73, K74, K75, K76, K77, K78, \
+	K80, K81, K82, K83, K84, K85  \
```

Comment:
This should be reorganized so that it resembles an actual keyboard layout. 

You can see a reference on how to do that here: 
https://docs.qmk.fm/#/understanding_qmk?id=matrix-to-physical-layout-map

Reply:
Hi, afaik this file shows what my matrix is like, in which case this would be correct as I used a fairly convoluted 9x9 matrix (with some empty spots) in order to fit all 78 keys on the Pro Micro. With that being said, is there anything else I should change about it? Thanks!

Reply:
What the matrix is actually doesn't matter here.  The point of the layout macro is so you can take crazy matrices and make them match what actual keyboard layout looks like.  

You can see an example of this with the Clueboard: 
https://github.com/qmk/qmk_firmware/blob/2d8703f6cc61918aaefd3ce04a064a82ab238140/keyboards/clueboard/66/rev4/rev4.h#L32-L49

Reply:
Thanks, this is good to know. I will for sure keep this in mind in the future :)